schema/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express'
+var app = express()
+app.use(express.json())
+
+// define a route handler for the default home page
+app.post('/fastGen', (req: express.Request, res: express.Response) => {

```

Comment:
pls rename routes to `/suggestion/fast` and `/suggestion/detailed` or something like that